scout/test/pageflow/page_load_status.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python
+"""
+Tests pages and their respective URL status codes
+"""
+
+import sys
+from django.test import TestCase
+
+
+class UrlStatusTest(TestCase):
+
+    def clientUrlStatus(self, urlsuffix=''):
+        """Returns the status code of the given URL"""
+        res = self.client.get(urlsuffix)
+        return res.status_code
+
+    def assertUrlStatus(self, urlsuffix='', code=200):
```

Comment:
I'd have to disagree, since it keeps it consistent with all the testcase.assert* methods, and test methods already have to start with test_, but your call